src/instance.ts

```diff
@@ -193,6 +249,264 @@ class Instance extends common.GrpcServiceObject {
+   * @param {GetBackupsOptions} [options] Contains query object for listing
+   *     backups and request configuration options, outlined here:
+   *     https://googleapis.github.io/gax-nodejs/CallSettings.html.
+   * @returns {Promise<GetBackupsResponse>} when resolved, contains a paged list of backups.
+   *
+   * @example
+   * const {Spanner} = require('@google-cloud/spanner');
+   * const spanner = new Spanner();
+   * const instance = spanner.instance('my-instance');
+   * const [backups] = await instance.getBackups();
+   */
+  getBackups(
+    optionsOrCallback?: GetBackupsOptions | GetBackupsCallback,
+    cb?: GetBackupsCallback
+  ): void | Promise<GetBackupsResponse> {
+    const callback =
+      typeof optionsOrCallback === 'function'
+        ? (optionsOrCallback as GetBackupsCallback)
+        : cb!;
+    const options =
+      typeof optionsOrCallback === 'object'
+        ? (optionsOrCallback as GetBackupsOptions)
+        : {gaxOptions: {}};
+    const gaxOpts: CallOptions = options.gaxOptions as CallOptions;
+    const reqOpts = extend({}, options, {
+      parent: this.formattedName_,
+    });
+    delete reqOpts.gaxOptions;
+
+    this.request<
+      IBackup,
+      databaseAdmin.spanner.admin.database.v1.IListBackupsResponse
+    >(
+      {
+        client: 'DatabaseAdminClient',
+        method: 'listBackups',
+        reqOpts,
+        gaxOpts,
+      },
+      (err, backups, ...args) => {
+        let backupInstances: Backup[] | null = null;
+        if (backups) {
+          backupInstances = backups.map(backup => {
+            return this.backup(backup.name!);
+          });
+        }
+
+        callback(err, backupInstances, ...args);
+      }
+    );
+  }
+
+  getBackupOperations(
+    options?: GetBackupOperationsOptions
+  ): Promise<GetBackupOperationsResponse>;
+  getBackupOperations(callback: GetBackupOperationsCallback): void;
+  getBackupOperations(
+    options: GetBackupOperationsOptions,
+    callback: GetBackupOperationsCallback
+  ): void;
+  /**
+   * Query object for listing backup operations.
+   *
+   * @typedef {object} GetBackupOperationsRequest
+   * @property {boolean} [autoPaginate=true] Have pagination handled
+   *     automatically.
+   * @property {number} [maxApiCalls] Maximum number of API calls to make.
+   * @property {number} [maxResults] Maximum number of items to return.
+   * @property {number} [pageSize] Maximum number of results per page.
+   * @property {string} [pageToken] A previously-returned page token
+   *     representing part of the larger set of results to view.
+   */
+  /**
+   * @typedef {array} GetBackupOperationsResponse
+   * @property {IOperation[]} 0 Array of {@link IOperation} instances.
+   * @property {object} 1 The full API response.
+   */
+  /**
+   * List pending and completed backup operations for all databases in the instance.
+   *
+   * @see {@link #listOperations}
+   *
+   * @param {GetBackupOperationsOptions} [options] Contains query object for
```

Comment:
This description is for `options.gaxOptions`. `options` is: https://github.com/prunge-helix/nodejs-spanner/blob/prunge/backups-samples-2020-premerge/src/instance.ts#364